### PR TITLE
py3: dnssec

### PR DESCRIPTION
**Pull request by tomaskrizek**
This PR is a partial fix that should allow DNSSEC installation for master. Keys will not be distributed to replicas. With my limited DNSSEC/IPA knowledge, I wasn't able to verify the data stored in LDAP are actually correct. In case they are not, this would prevent installation of DNSSEC replicas in the future. 

Our DNSSEC tests are not passing, thus we can't use them to verify this PR.  Given these circumstances, I propose to officially discourage DNSSEC installation in 4.6.0.

This PR supersedes #898. For review, it is highly recommended to rebase on #999.

### CODE REVIEW

**Comment by tomaskrizek** [comment]
As we agreed on the devel mtg, we'll keep DNSSEC in Python 2 for now.

**Comment by stlaz** [comment]
You'll need to rebase on the current master for the tests to pass.

**Comment by stlaz** [comment]
Seems to have been closed by an accident.

File: ipaserver/dnssec/bindmgr.py [hunk 1]
```diff
@@ -141,17 +145,21 @@ def get_zone_dir_name(self, zone):
...
-                    escaped += chr(c)
+                if six.PY2:
+                    # PY3 char is already int
+                    char = ord(char)
+                if (
+                    (char >= 0x30 and char <= 0x39) or  # digit
+                    (char >= 0x41 and char <= 0x5A) or  # uppercase
+                    (char >= 0x61 and char <= 0x7A) or  # lowercase
+                    char == 0x2D or                     # hyphen
+                    char == 0x5F                        # underscore
```
Comment (tiran) [inline ipaserver/dnssec/bindmgr.py]: ETOOMUCHC

```
if six.PY3:
    char = chr(char)
if char.isalnum() or char in "-_":
    escaped += char.lower()
else:
    escaped += "%%%02X" % ord(char)
```

**Comment by tomaskrizek** [comment]
If someone wants to take over this PR, feel free to do so. I think DNSSEC test suite should be fixed first (#973) so we can at least somehow verify the functionality.

**Comment by tiran** [comment]
Closed in favor of new PR #1289

**Pull request closed without merging**